### ISSUE TITLE
[REQUEST] Interacting with telegram plugin to notify the user

### ISSUE THREAD

**Issue by MasterPIC** [kantlivelong/OctoPrint-UPS#9]
Hi,

what about making use of

**M118 E1 UserNotif UPS on battery**
and
**M118 E1 UserNotif UPS online**

to make Telegram notifying the user when UPS switches to/from battery?

Thank you for this really nice plugin!
Armando

**Comment by kantlivelong** [comment]
I agree that some kind of notification would be useful though I'm not sure how exactly I want to implement it at this time. When I do it will be a more generic approach.

**Comment by ripcurl7777** [comment]
Maybe there is an Option over the Plugin OctoPodNotifications
